main.js

```diff
@@ -249,7 +249,22 @@ function addAll() {
 /* 13. Pass in a number to loop up to and add all of the prime numbers. A prime number is a whole number greater than 1 whose only factors are 1 and itself
  * ex. sumAllPrimes(10) == 17
  */
-function sumAllPrimes() {}
+function sumAllPrimes() {
+  let sum = 0;
```

Comment:
try building the **sieve of eratosthenes** in JS, and use it to solve this problem